app/javascript/components/widgets/widgets/forest-change/tree-loss-located/selectors.js

```diff
@@ -107,6 +107,16 @@ export const getSentence = createSelector(
     }
     const topLoss = percentileLoss / totalLoss * 100 || 0;
     let sentence = !indicator ? initialPercent : withIndicatorPercent;
+    const value =
+      topRegion.percentage > 0 && settings.unit === '%'
+        ? `${format('.2r')(topRegion.percentage)}%`
```

Comment:
Again here. You could do `const formatType = topRegion.percentage > 0 ? '.2r' : '.3s'`